modules/deploy/pages/deployment-option/self-hosted/kubernetes/kubernetes-deploy.adoc

```diff
@@ -86,11 +102,17 @@ spec:
   clusterSpec: {}
 ----
 +
+- `metadata.name`: Name to assign the Redpanda cluster. This name is also assigned to the Helm release.
+- xref:reference:crd.adoc#chartref[`spec.chartRef`]: Information about the Helm chart that will be used to deploy Redpanda.
```

Comment:
```suggestion
- xref:reference:crd.adoc#{anchor_prefix}-github-com-redpanda-data-redpanda-src-go-k8s-apis-redpanda-v1alpha1-chartref[`spec.chartRef`]: Information about the Helm chart that will be used to deploy Redpanda.
```
I'm actually not 100% sure about the syntax here, but clicking on the link (at least in the deploy preview) only takes you near the beginning of the CRD reference and not the actual section. In the crd.adoc file the ID is what I've pasted above ^